### DISCUSSION TITLE
I don't have root access to the ES cluster. I set the index template before using ES, but now I get an error. How should I solve it?

### DISCUSSION THREAD

**Discussion by wallywang110** [apache/skywalking#11556] [Q&A]
As mentioned, the error message is as follows:
`2023-11-16 15:19:52,185 - org.apache.skywalking.library.elasticsearch.client.TemplateClient - 139 [armeria-eventloop-epoll-5-3] ERROR [] - Failed to create / update index template ht-sw_records-all
java.util.concurrent.CompletionException: java.lang.RuntimeException: {"error":{"root_cause":[{"type":"security_exception","reason":"action [indices:admin/index_template/put] is unauthorized for user [htgroup] with roles [htgroup], this action is granted by the cluster privileges [manage_index_templates,manage,all]"}],"type":"security_exception","reason":"action [indices:admin/index_template/put] is unauthorized for user [htgroup] with roles [htgroup], this action is granted by the cluster privileges [manage_index_templates,manage,all]"},"status":403}`

**Comment by wu-sheng** [comment]
We dont' have that scripts. You could set up a local elasticsearch, and boot the OAP server. Then export all created templates and indices into your target.
But notice, your account to create templates should have authority to trigger index auto creation.
